MAVProxy/modules/mavproxy_rfsurvey.py

```diff
@@ -340,7 +343,7 @@ def close_csv_fh(self):
         self.csv_fh = None
 
     def open_new_csv_fh(self):
-        filename = "rfsurvey-%s.csv" % (time.strftime("%Y%m%d%H%M%s", time.localtime()),)
+        filename = "rfsurvey-temp.csv"
```

Comment:
This will overwrite the same file each time you start/stop the module.

This could easily lose important data.

Was this intentional?